src/render.ts

```diff
@@ -164,408 +185,288 @@ export class ImageRenderer {
                 },
                 outline: {
                     name: 'off', params: {}
+                },
+                antialiasing: {
+                    name: 'off', params: {}
                 }
             }
-        })
+        });
         this.imagePass = this.canvas3d.getImagePass({
-            drawPass: {
-                cameraHelper: {
-                    axes: { name: 'off', params: {} }
-                }
+            cameraHelper: {
+                axes: { name: 'off', params: {} }
             },
             multiSample: {
                 mode: 'on',
                 sampleLevel: 3
```

Comment:
I would use 4

Reply:
I've updated all params to your recommendations.